ydb/docs/en/core/reference/ydb-sdk/topic.md

```diff
@@ -1382,6 +1382,8 @@ If a commit fails with an error, the application should log it and continue; it
    }
    ```
 
+   The `Commit` call is fast by default, saving data into an internal buffer and returning control back to the caller. The real message to the server is sent in the background. To prevent losing the last commits, you should call the `Reader.Close()` method before exiting the program.
```

Comment:
```suggestion
   The `Commit` call is fast by default, saving data into an internal buffer and returning control to the caller. The real message to the server is sent in the background. To prevent losing the last commits, call the `Reader.Close()` method before exiting the program.
```
Minor, fewer words with the same meaning.